cmd/dockerd/daemon_darwin.go

```diff
@@ -0,0 +1,11 @@
+// +build darwin
+
```

Comment:
maybe renaming cmd/dockerd/daemon_freebsd.go to cmd/dockerd/daemon_unix.go would decrease the patch size ?

Reply:
I don't think so. There already exists something like a daemon_unix.go in cmd/dockerd/. And it isn't handling preNotifySystem(), notifySystem() these functions. For these, they have separate files like _linux, _freebsd, etc. I didn't wanted to mix up the existing architecture much or existing code. If you think we should add this functions to daemon_unix.go and delete each separate  _linux, _freebsd, _darwin.go - I can do it. But that's not gonna decrease the patch size.

Reply:
Okay. I confirmed.